internal/wire/subscribe_message.go

```diff
@@ -78,34 +79,34 @@ func (m *SubscribeMessage) LogValue() slog.Value {
 }
 
 func (m SubscribeMessage) GetSubscribeID() uint64 {
```

Comment:
Should probably be GetRequestID

Reply:
I removed the method, as it is no longer used (was a leftover from a long time ago, I think.)